docs/learn/getting-started/prerequisites.md

```diff
@@ -0,0 +1,346 @@
+```
+
+### Linux (Ubuntu/Debian)
+
+```bash
+# Install Node.js via NodeSource
+curl -fsSL https://deb.nodesource.com/setup_20.x | sudo -E bash -
+sudo apt-get install -y nodejs
+
+# Verify installation
+node --version
+npm --version
+```
+
+### Windows
+
+Download and run the installer from [nodejs.org](https://nodejs.org/).
+
+---
+
+## Step 2: Install Core Dependencies
+
+Create a new project and install the required packages:
+
+```bash
+# Create project directory
+mkdir yellow-app && cd yellow-app
+
+# Initialize project
+npm init -y
+
+# Install core dependencies
+npm install @erc7824/nitrolite viem
+
+# Install development dependencies
+npm install -D typescript @types/node tsx
+```
+
+### Package Overview
+
+| Package | Purpose |
+|---------|---------|
+| `@erc7824/nitrolite` | Yellow Network SDK for state channel operations |
+| `viem` | Modern Ethereum library for wallet and contract interactions |
+| `typescript` | Type safety and better developer experience |
+| `tsx` | Run TypeScript files directly |
+
+---
+
+## Step 3: Configure TypeScript
+
+Create `tsconfig.json`:
+
+```json
+{
+  "compilerOptions": {
+    "target": "ES2022",
+    "module": "ESNext",
+    "moduleResolution": "bundler",
+    "strict": true,
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "outDir": "./dist",
+    "rootDir": "./src"
+  },
+  "include": ["src/**/*"],
+  "exclude": ["node_modules"]
+}
+```
+
+Update `package.json`:
+
+```json
+{
+  "type": "module",
+  "scripts": {
+    "dev": "tsx watch src/index.ts",
+    "build": "tsc",
+    "start": "node dist/index.js"
+  }
+}
+```
+
+---
+
+## Step 4: Set Up Environment Variables
+
+Create `.env` for sensitive configuration:
+
+```bash
+# .env - Never commit this file!
+
+# Your wallet private key (for development only)
+PRIVATE_KEY=0x...
+
+# RPC endpoints
+SEPOLIA_RPC_URL=https://sepolia.infura.io/v3/YOUR_KEY
+BASE_RPC_URL=https://base-sepolia.g.alchemy.com/v2/YOUR_KEY
+
+# Clearnode WebSocket endpoint
+CLEARNODE_WS_URL=wss://clearnode.yellow.com/ws
+
+# Contract addresses (get from Yellow Network docs)
+CUSTODY_ADDRESS=0x...
+ADJUDICATOR_ADDRESS=0x...
+```
+
+Add to `.gitignore`:
+
+```bash
+# .gitignore
+.env
+.env.local
+node_modules/
+dist/
+```
+
+Install dotenv for loading environment variables:
+
+```bash
+npm install dotenv
+```
+
+---
+
+## Step 5: Wallet Setup
+
+### Development Wallet
+
+For development, create a dedicated wallet:
+
+```typescript
+// scripts/create-wallet.ts
+import { generatePrivateKey, privateKeyToAccount } from 'viem/accounts';
+
+const privateKey = generatePrivateKey();
+const account = privateKeyToAccount(privateKey);
+
+console.log('New Development Wallet');
+console.log('----------------------');
+console.log('Address:', account.address);
+console.log('Private Key:', privateKey);
+console.log('\n⚠️  Save this private key securely and add to .env');
+```
+
+Run it:
+
+```bash
+npx tsx scripts/create-wallet.ts
+```
+
+### Get Test Tokens
```

Comment:
Test tokens are available only on Sandbox deployment. The following command can be used to request `ytest.USD` on user's unified balance:
```
curl -XPOST https://clearnet-sandbox.yellow.com/faucet/requestTokens -d '{"userAddress":"<user_address>"}'
```

Once requested, you won't need to perform any off-chain operation as tokens are already on unified balance.